src/chunk.c

```diff
@@ -5143,3 +5144,26 @@ get_chunks_in_creation_time_range(Hypertable *ht, int64 older_than, int64 newer_
 
 	return chunks;
 }
+
+Datum
+ts_chunk_drop_osm_chunk(PG_FUNCTION_ARGS)
+{
+	Oid hypertable_relid = PG_ARGISNULL(0) ? InvalidOid : PG_GETARG_OID(0);
+	Cache *hcache = ts_hypertable_cache_pin();
+	Hypertable *ht = ts_resolve_hypertable_from_table_or_cagg(hcache, hypertable_relid, true);
+	int32 osm_chunk_id = ts_chunk_get_osm_chunk_id(ht->fd.id);
+	Chunk *osm_chunk = ts_chunk_get_by_id(osm_chunk_id, true);
+
+	ts_chunk_validate_chunk_status_for_operation(osm_chunk, CHUNK_DROP, true);
+
+	/* do not drop any chunk dependencies */
+	ts_chunk_drop(osm_chunk, DROP_RESTRICT, LOG);
```

Comment:
What are the dependencies in this case, and why shouldn't we drop them? Might be good to extend the comment.